wasm2c/wasm-rt-impl.c

```diff
@@ -206,6 +212,31 @@ static void os_print_last_error(const char* msg) {
     printf("%s. No error code.\n", msg);
   }
 }
+
+#if WASM_RT_MEMCHECK_SIGNAL_HANDLER && !WASM_RT_SKIP_SIGNAL_RECOVERY
+
+static LONG os_signal_handler(PEXCEPTION_POINTERS pExceptionInfo) {
+  if (pExceptionInfo->ExceptionRecord->ExceptionCode ==
```

Comment:
I suppose this is windows-only code, but maybe we should use the wabt naming convension here?   i.e. call this argument just `info` maybe?   (that might allow the conditions below to fit on a single line too).

Reply:
will fix